src/darker/tests/test_import_sorting.py

```diff
@@ -55,7 +55,43 @@ def test_apply_isort(git_repo, encoding, newline, content, expect):
     src = Path("test1.py")
     content_ = TextDocument.from_lines(content, encoding=encoding, newline=newline)
 
-    result = darker.import_sorting.apply_isort(content_, src, edited_linenums_differ)
+    result = darker.import_sorting.apply_isort(
+        content_, src, exclude=[], edited_linenums_differ=edited_linenums_differ
+    )
+
+    assert result.lines == expect
+    assert result.encoding == encoding
+    assert result.newline == newline
+
+
+@pytest.mark.parametrize("encoding", ["utf-8", "iso-8859-1"])
+@pytest.mark.parametrize("newline", ["\n", "\r\n"])
```

Comment:
maybe add `\r` from Mac? `["\n", "\r\n", "\r"]`

Reply:
I started adding such test cases (there are lots of these across the test suite), and started getting test failures. See the [mac-newline-tests](akaihola/darker/tree/mac-newline-tests) branch.

However, I haven't yet heard of any complaints from macos users about newlines being treated wrong. Are you sure about `\r`?